### PR TITLE
webp lossless: Implement pixel bundling for color indexing transform

### PR DESCRIPTION
**Pull request by nico**
With this, the webp lossless decoder is feature complete :^)

Of course, this last part of the last missing feature invalidated a few assumptions I had made – and the spec isn't super clear about these assumptions not being true. But it wasn't too bad.

### CODE REVIEW

**Review by linusg — COMMENTED** [review]
Typo in third commit: `peformance`

**Comment by nico** [comment]
Fixed…but the ALL_DEBUG bot also was red I think. (Sadly, it disappeared when I pushed and I couldn't look.) So let's wait a bit for what that bot says.

File: Userland/Libraries/LibGfx/ImageFormats/WebPLoader.cpp [hunk 1]
```diff
@@ -755,7 +757,7 @@ Transform::~Transform() = default;
 class PredictorTransform : public Transform {
 public:
     static ErrorOr<NonnullOwnPtr<PredictorTransform>> read(WebPLoadingContext&, LittleEndianInputBitStream&, IntSize const& image_size);
-    virtual ErrorOr<void> transform(Bitmap&) override;
+    virtual ErrorOr<NonnullRefPtr<Bitmap>> transform(NonnullRefPtr<Bitmap>);
```
Comment (linusg) [inline Userland/Libraries/LibGfx/ImageFormats/WebPLoader.cpp:760]:
```suggestion
    virtual ErrorOr<NonnullRefPtr<Bitmap>> transform(NonnullRefPtr<Bitmap>) override;
```

(should fix CI build error, too)

**Review by linusg — COMMENTED** [review]
Looks good but CI fell over, not sure how to fix _that_

**Comment by nico** [comment]
I rebased and repushed, maybe that'll help. (CI is pretty flaky in general.)

**Comment by nico** [comment]
This time "Wasm Modules" came back green.

**Pull request merged**